packages/lib/hooks/useTypedQuery.ts

```diff
@@ -57,7 +57,8 @@ export function useTypedQuery<T extends z.AnyZodObject>(schema: T) {
   // Delete a key from the query
   function removeByKey(key: OutputOptionalKeys) {
     const { [key]: _, ...newQuery } = parsedQuery;
-    router.replace({ query: newQuery as Output }, undefined, { shallow: true });
+    const search = new URLSearchParams(newQuery).toString();
+    router.replace({ query: search }, undefined, { shallow: true });
```

Comment:
It was causing `teamid=[1,2]` to become `teamId=1&teamId=2`
cc @sean-brydon

Reply:
Interesting okay - happy with this aslong as it works here and elsewhere 

(will test)